dev/com.ibm.ws.security.jwtsso_fat.commonTest/fat/src/com/ibm/ws/security/jwtsso/fat/ReplayCookieTests.java

```diff
@@ -293,13 +300,11 @@ public void test_reaccessResource_signatureRemovedFromJwtCookie() throws Excepti
     @Test
     public void test_obtainLtpa_reconfigureToUseJwtSso_reaccessResourceWithLtpaCookie() throws Exception {
 
-        server.removeInstalledAppForValidation(APP_NAME_JWT_BUILDER);
-        server.reconfigureServerUsingExpandedConfiguration(_testName, "server_noFeature.xml");
+        expandAndUpdateServerConfiguration(server, "server_noFeature.xml", true, "formlogin", "testmarker");
```

Comment:
Could you use the constants for "formlogin" and "testmarker"? 
JwtFatConstants.APP_FORMLOGIN and JwtFatConstants.TESTMARKER

Reply:
Done, thank you